src/App.tsx

```diff
@@ -1,61 +1,163 @@
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+function getUser(id: number) {
+  return usersFromServer.find(person => id === person.id);
+}
+
+const todosWithUser = todosFromServer.map(todo => {
+  const user = getUser(todo.userId);
+
+  return {
+    ...todo,
+    user,
+  };
+});
+
+function getId(todos: Todo[]) {
+  const ids = todos.map(todo => todo.id);
+  const newId = Math.max(...ids) + 1;
+
+  return newId;
+}
 
 export const App = () => {
+  const [todos, setTodos] = useState(todosWithUser);
+  const [formValues, setFormValues] = useState({
+    todoTitle: '',
+    todoUser: '0',
+  });
+  const [formErrors, setFormErrors] = useState({
+    titleError: false,
+    userError: false,
+  });
```

Comment:
I found this way of using an object as a variable container for controlling `useState` very interesting, but also quite difficult))